content/pages/privacy-policy.md

```diff
@@ -0,0 +1,93 @@
+
+The 3rd party that can be contacted by Mixxx, automatically is:
+
+ - MusicBrainz - For this service, please refer their Privacy Policy here: [https://metabrainz.org/privacy](https://metabrainz.org/privacy)
+
+### Broadcasting
+
+If you use the broadcasting feature of the Mixxx application, you need to enter login data of a third party Soundcast/Icecast server.
+Mixxx itself doesn't offer such a service. Only metadata of the played tracks are send to these servers by the Mixxx application.
+
+### Mixxx Crash Reporting
+
+The Mixxx application itself neither collects crash reports nor shares them.
+
+Crash reports are handled by the Stores/Vendors of each platforms.
+Please refer to those platforms for information about the privacy policies.
+Depending on the platform, Mixxx Core Developers have access to these reports, which usually contain:
+
+- Operating System version
+- CPU and GPU
+- Symbolic StackTrace of the process
+- List of DLLs loaded by the process
+
+None of these data is stored on Mixxx servers.
+In addition Mixxx writes a mixxx.log file which contains entries that may expose personal data. In a support case we may ask to share this file in public. This is a manual process during which you may obfuscate sensitive date before sharing.  
+### Mixxx Update Mechanism
+
+The Mixxx application itself doesn't provide an update mechanism.
+
+In case updates are handled by the platform stores, where the user has downloaded Mixxx from,
+please refer to those platforms for information about the privacy policies.
+
+
+## 2. Main Mixxx Website
+
+The main website of Mixxx, aka www.mixxx.org, does not create or use any user account.
+There are no registration forms, or any similar mechanism.
+Therefore, no personal information is stored at any moment, about the user, by Mixxx.
+
+## 3. Community Mixxx Websites
+
+The community websites from Mixxx are related to development and community, and are the following:
+
+- GitHub (Code, Issue Tracking, Wiki)
+- Discourse Forum
+- Zulip Chat
+
+Each of those websites have different users system, but work in a similar way. User need to log in and have a profile, this requires the use of login data and cookies. Refer the privacy statements of these sites for details:
+
+- You can find the GitHub Privacy Statement here: [https://docs.github.com/de/site-policy/privacy-policies/github-privacy-statement](https://docs.github.com/de/site-policy/privacy-policies/github-privacy-statement)
+- You can find the Privacy Policy of the Mixxx Discourse Forum here: [https://mixxx.discourse.group/privacy](https://mixxx.discourse.group/privacy)
+- You can find the Zulip Privacy Policy here: [https://zulip.com/policies/privacy](https://zulip.com/policies/privacy)
+
+One can modify their user information at any time, through the options of those websites.
+
+## Contact Mixxx
+
+Various contact options are listed here: [Contact Mixxx](contact)

```

Comment:
The link is broken. This should work:

```suggestion
Various options to get in touch are listed on the [contact page]({filename}/contact.md). 
```

Reply:
This doesn't work

Reply:
Sorry, accidently used an absolute link where a relative one should be used (`{filename}contact.md` instead of `{filename}/contact.md` works fine). The current solution is brittle. If we rename the contact page or change the path structure this will stop working and pelican will not be able to warn us about the broken link.

Also, the current sentence sounds awkward to me.

Reply:
It's fixed already - using `/contact`

Reply:
I know, this is why I elaborated why it's still a good idea to [use the proper way to link to internal content](https://docs.getpelican.com/en/latest/content.html#linking-to-internal-content):

> The current solution is brittle. If we rename the contact page or change the path structure this will stop working and pelican will not be able to warn us about the broken link.

Reply:
Changed